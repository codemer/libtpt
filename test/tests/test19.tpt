@include("test19inc.tfh")\
@header("My Title")\
This is the body of my HTML
@footer()\
