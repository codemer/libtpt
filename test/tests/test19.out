<html>
<head><title>My Title</title></head>
<body>
<h1>My Title</h1>
<hr>
This is the body of my HTML
</body>
</html>
